### PR TITLE
[Fix] Empty inventory data in Solaris agents

### PR DESCRIPTION
**Pull request by gabiwassan**
Hi team!
### Description
- When entering Inventory Data for a Solaris agent, the entire view was blank. 
- The necessary validations were carried out, to show all fields correctly, with their data empty.
eg: Solaris agent:
image
eg: Ubuntu agent:
image

Issue: #2484

### CODE REVIEW

File: public/components/agents/syscollector/components/syscollector-table.tsx [hunk 1]
```diff
@@ -117,19 +116,25 @@ export function SyscollectorTable({ tableParams }) {
...
-          <EuiFlexItem grow={true} > </EuiFlexItem>
-          <EuiFlexItem grow={false} >
+      {tableParams.exportFormatted && tableParams.columns && (
+        <EuiFlexGroup>
+          <EuiFlexItem grow={true} />
+          <EuiFlexItem grow={false}>
             <EuiButtonEmpty
-              onClick={async () => await AppState.downloadCsv(tableParams.path, tableParams.exportFormatted, !!params.search ? [{ name: 'search', value: params.search }] : [])}
-              iconType="importAction">
+              onClick={async () =>
```
Comment (frankeros) [inline public/components/agents/syscollector/components/syscollector-table.tsx:124]: better if this is in a separate function

Comment (gabiwassan) [reply public/components/agents/syscollector/components/syscollector-table.tsx:124]: Do you say the onClick event?

**Review by frankeros — APPROVED** [review]
LGTM!

**Comment by Desvelao** [comment]
I was testing the PR and the tables keep empty as the screenshot of the PR comment.

You should add the issue's number and URL to the changelog file.

**Review by Desvelao — COMMENTED** [review]
LGTM!

**Review by Desvelao — APPROVED** [review]

**Pull request merged**